lib/api/v3/work_packages/form/work_package_payload_representer.rb

```diff
@@ -64,6 +64,13 @@ def initialize(represented, options = {})
 
           property :lock_version
           property :subject, render_nil: true
+          property :done_ratio,
+                   as: :percentageDone,
+                   getter: -> (*) { done_ratio if Setting.work_package_done_ratio != 'disabled' },
+                   render_nil: false
+          property :estimated_hours,
```

Comment:
The estimated time should be formatted and accepted as ISO 8601 duration only.

Currently you are presenting and accepting the raw float value.